### ISSUE TITLE
Test does not resume after pausing/waiting for more than 13 seconds

### ISSUE THREAD

**Issue by sarahncrowe** [DevExpress/testcafe#3070]
### What is your Test Scenario?
I'm testing a timer component in which I start a timer and pause the test until 60 seconds has passed, at which point I stop the timer and expect the duration to be saved.
 
### What is the Current behavior?
The test doesn't continue after pausing for more than 13000 ms in Chrome and Firefox.
 
### What is the Expected behavior?
Resume and execute the rest of the test
 
### What is your web application and your TestCafe test code?

<details>
<summary>Your complete test code (or attach your test files):</summary>

 I've added the debug so that it's more obvious when the wait finishes, but the test still doesn't continue without it

``` javascript
fixture `Example`
    .page('https://www.google.ca');

test('Test does not resume after pausing', async t => {
    await t
        .expect(await Selector('#searchform').exists).ok();

    await t
        .wait(15000);

    await t
        .debug()
        .expect(await Selector('#searchform').exists).ok();
});
```
</details>

<details>
<summary>Your complete test report:</summary>
The test does not finish, so no report
</details>

<details>
<summary>Screenshots:</summary>

![image](https://user-images.githubusercontent.com/6461649/47864780-d9188900-ddd0-11e8-8864-7ad7c4504331.png)

![image](https://user-images.githubusercontent.com/6461649/47866060-2f3afb80-ddd4-11e8-9166-89ea7eab37fb.png)

</details>
 
### Your Environment details:
 
* testcafe version:                   0.23.0
* node.js version:                    8.11.3
* command-line arguments:  testcafe chrome,firefox example.js
* browser name and version: Chrome 70, Firefox 62
* platform and version:          Windows 10.0
* other:                                   IE 11 seemed to work correctly with a wait of 20000 ms

**Comment by AlexKamaev** [comment]
I was able to reproduce the issues. We need to research it

**Comment by link89** [comment]
We suffered this problem either. Hope could be fixed soon.

**Comment by link89** [comment]
Hi @miherlosev , this fix will be included in which stable version?

**Comment by LavrovArtem** [comment]
The fix was published in `testcafe@0.23.3-alpha.1`.
